libs/remix-ui/statusbar/src/lib/components/scamDetails.tsx

```diff
@@ -30,16 +30,16 @@ export default function ScamDetails ({ refs, floatStyle, scamAlerts }: ScamDetai
       } }
       className="p-1 pb-0 mb-0 d-flex alert alert-warning border border-warning"
     >
-      <span className="align-self-center pl-4 mt-1">
+      <span className="align-self-center pl-2 mt-1">
         <i style={{ fontSize: 'xxx-large', fontWeight: 'lighter' }} className="pr-2 far fa-exclamation-triangle"></i>
       </span>
-      <div className="d-flex flex-column pr-4 py-2">
+      <div className="d-flex flex-column pr-2 py-2">
         {scamAlerts && scamAlerts.map((alert, index) => (
-          <span className="pl-4 mt-1" key={`${alert.url}${index}`}>
+          <span className="pl-2 mt-1" key={`${alert.url}${index}`}>
             {alert.url.length < 1 ? <FormattedMessage id={`home.scamAlertText${index + 1}`} defaultMessage={alert.message} />
-              : (<><FormattedMessage id={`home.scamAlertText${index + 1}`} defaultMessage={alert.message} /> : &nbsp;
+              : (<><FormattedMessage id={`home.scamAlertText${index + 1}`} defaultMessage={alert.message} /> :
                 <a
-                  className={`remixui_home_text text-decoration-none ${index === 1 ? 'pl-2' : ''}`}
+                  className={`remixui_home_text text-decoration-none pl-1`}
```

Comment:
isn't the index supposed to do something?

Reply:
It was adding extra space for first link like this:

<img width="573" alt="Screenshot 2024-12-24 at 3 30 31 PM" src="https://github.com/user-attachments/assets/a09a93b9-22b9-447d-9b0c-1c21105cf616" />


 don't see a requirement of that: